File: Beginner/go/1060.go
// 
package main

import "fmt"

func main() {
	var t, i, count int
	t = 6
	count = 0
	var n float32
	for i = 0; i < t; i++{
		fmt.Scan(&n)
		if n > 0{
			count++
		}
	}
	fmt.Printf("%d valores positivos\n", count)
}

File: Beginner/cpp/1060.cpp
// Write a program 

#include <bits/stdc++.h>
using namespace std;

int main() {
    int t = 6, count = 0;
    double n;

    while(t--){
        scanf("%lf", &n);
        if(n > 0) count++;
    }
    // printf("%d valores positivos\n", count);
    cout << count << " valores positivos\n";

return 0;
}